tick/base/array_test/src/array_test.cpp

```diff
@@ -27,18 +27,17 @@ double test_constructor_ArrayDouble(unsigned long size){
     return a_i * b_i * c[2] * d[3];
 }
 
-SArrayDoublePtr test_arange(long min, long max) {
+SArrayDoublePtr test_arange(std::int64_t min, std::int64_t max) {
```

Comment:
long is not authorized? I thought only unsigned long was. Don't we use long in many other places?